File: Strings/1873.py
n = int(input())

for i in range(n):
    linha = input().split()
    r = str(linha[0])
    s = str(linha[1])

    if r == s:
        print('empate')
    elif (r == 'tesoura' and s == 'papel') or (r == 'tesoura' and s == 'lagarto'):
        print('rajesh')
    elif (r == 'papel' and s == 'pedra') or (r == 'papel' and s == 'spock'):
        print('rajesh')
    elif (r == 'pedra' and s == 'lagarto') or (r == 'pedra' and s == 'tesoura'):
        print('rajesh')
    elif (r == 'spock' and s == 'tesoura') or (r == 'spock' and s == 'pedra'):
        print('rajesh')
    elif (r == 'lagarto' and s == 'spock') or (r == 'lagarto' and s == 'papel'):
        print('rajesh')
    else:
        print('sheldon')

File: Strings/1873.js
let nTestes = Number(lines.shift());

for (let i = 0; i < nTestes; i++) {
  let [r, s] = lines.shift().split(' ');
  
  if (r === s) {
    console.log('empate');
  } else if (r == 'tesoura' && s == 'papel' || r == 'tesoura' && s == 'lagarto') {
    console.log('rajesh');
  } else if (r == 'papel' && s == 'pedra' || r == 'papel' && s == 'spock') {
    console.log('rajesh');
  } else if (r == 'pedra' && s == 'lagarto' || r == 'pedra' && s == 'tesoura') {
    console.log('rajesh');
  } else if (r == 'spock' && s == 'tesoura' || r == 'spock' && s == 'pedra') {
    console.log('rajesh');
  } else if (r == 'lagarto' && s == 'spock' || r == 'lagarto' && s == 'papel') {
    console.log('rajesh');
  } else {
    console.log('sheldon');
  }
}
